e2e/terraform/compute.tf

```diff
@@ -98,10 +100,10 @@ data "aws_ami" "ubuntu_jammy_amd64" {
     values = ["Ubuntu"]
   }
 
-  filter {
+  /* filter {
```

Comment:
This filter was removed for testing, but should we restore it for the merge to main?

Reply:
right, my bad